### PR TITLE
chore(dep): Upgrade react-dev-utils to support webpack5 (#67)

### PR DESCRIPTION
**Pull request by chang-ke**

### CODE REVIEW

**Comment by nikrowell** [comment]
@chang-ke thanks for updating this! 
@gregberge or @jsfez - would love to see this PR get merged when you have time 😀

**Comment by chang-ke** [comment]
@gregberge

File: package.json [hunk 1]
```diff
@@ -37,8 +37,7 @@
   },
   "license": "MIT",
   "dependencies": {
-    "react-dev-utils": "^11.0.1",
-    "react-error-overlay": "^6.0.8",
```
Comment (ntucker) [inline package.json:41]: peerDeps need to be updated to include webpack 5

Comment (chang-ke) [reply package.json:41]: done

**Comment by bahtou** [comment]
any updates on this? would love to give this a go on my current setup

**Comment by DamianBusz** [comment]
Bumping this for visibility, still would love to see this merged

**Comment by gregberge** [comment]
Thanks!

**Comment by DamianBusz** [comment]
Thank you very much!

**Pull request merged**